### PR TITLE
Refactor node source reference section

### PR DESCRIPTION
**Pull request by fviale**
- create top level chapters for infrastructures and policies
 - harmonize formatting for configuration parameters
 - small text corrections
 - clarify the usage of cluster scheduler infrastructures (and policy)

### CODE REVIEW

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 1]
```diff
@@ -1,8 +1,8 @@
-An *Infrastructure Manager* is responsible for deploying nodes. In most of the cases it knows how to launch a node
+A *Node Source Infrastructure* also called *Infrastructure Manager* is responsible for deploying ProActive Nodes inside a defined <<_deploy_nodes_from_proactive_rm,*Node Source*>>. In most of the cases, it knows how to launch a set of ProActive Nodes
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: A *Node Source Infrastructure*, also called *Infrastructure Manager*, is responsible....  [missing commas]

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 2]
```diff
@@ -216,7 +216,7 @@ WARNING: If ProActive server is forcibly killed, the created AWS instances will
 
 
 
-===== Openstack Infrastructure
+==== Openstack Infrastructure
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: OpenStack   (official naming)

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 3]
```diff
@@ -296,7 +296,7 @@ WARNING: When ProActive server is stopped (without removing the created infrastr
 
 
 
-===== VmWare Infrastructure
+==== VmWare Infrastructure
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: VMware     (official naming)

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -2499,24 +2499,13 @@ include::./references/CatalogReference.adoc[]
...
-The ProActive Resource Manager supports ProActive Nodes aggregation from heterogeneous environments.
-As a node is just a process running somewhere, the process of communication to such nodes is unified.
-The only part which has to be defined is the procedure of nodes deployment which could be quite
-different depending on infrastructures and their limitations. After installation of the server and node parts it is
-possible to configure an automatic nodes deployment. Basically, you can say to the Resource Manager how to
-launch nodes and when.
-
-In the ProActive Resource Manager, there is always a default node source consisted of DefaultInfrastructureManager
-and Static policy. It is not able to deploy nodes anywhere but makes it possible to add existing nodes to the RM.
-
```
Comment (tao-xinxiu) [inline src/docs/admin/ProActiveAdminGuide.adoc]: I think this paragraph can be revised and moved into section 4 (Deploy ProActive Nodes) introduction.
For example, before the paragraph "A node source is defined by two parts of configurations..."

File: src/docs/admin/references/NodeSourceInfrastructureReference.adoc [hunk 5]
```diff
@@ -423,11 +423,11 @@ An GoogleComputeEngine NodeSource can now be added using the *Create Node Source
 WARNING: When ProActive server is stopped, GoogleComputeEngine instances will be automatically deleted. And when ProActive server is restarted, the infrastructure will be recovered as per the previous settings. The required GoogleComputeEngine instances will be re-created and re-configured.
 
 
-===== Load Sharing Facility (LSF) infrastructure
+==== Load Sharing Facility (LSF) infrastructure
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourceInfrastructureReference.adoc]: We should agree whether "infrastructure" is capitalized or not in the section titles, we use both lower and upper case so far.

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 6]
```diff
@@ -29,13 +29,13 @@ or +PROVIDER_GROUPS+ (unless it's granted AllPermissions).
...
 
-===== Static Policy
+==== Static Policy
 
 *Static node source policy* starts node acquisition when nodes are added
 to the node source and never removes them. Nevertheless, nodes can be
 removed by user request. To use this policy, you have to precise the following parameters:
 
--   +nodeUsers+ - utilization permission defined who can get nodes for
+-   *nodeUsers* - utilization permission defined who can get nodes for
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc]: Utilization    [capital U]

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 7]
```diff
@@ -59,12 +59,12 @@ removed by user request. To use this policy, you have to precise the following p
...
 
 
-===== Time Slot Policy
+==== Time Slot Policy
 
 *Time slot policy* is aimed to acquire nodes for particular time with an
 ability to do it periodically. To use this policy, you have to precise the following parameters:
 
--   +nodeUsers+ - utilization permission defined who can get nodes for
+-   *nodeUsers* - utilization permission defined who can get nodes for
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc]: we should do a replace all from "utilization permission" to "Utilization permission" (capital U)

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 8]
```diff
@@ -87,24 +87,24 @@ ability to do it periodically. To use this policy, you have to precise the follo
...
--   +acquireTime+ - Absolute acquire date (e.g. "6/3/10 1:18:45 PM
+-   *acquireTime* - Absolute acquire date (e.g. "6/3/10 1:18:45 PM
     CEST").
 
--   +releaseTime+ - Absolute releasing date (e.g. "6/3/10 2:18:45 PM
+-   *releaseTime* - Absolute releasing date (e.g. "6/3/10 2:18:45 PM
     CEST").
 
--   +period+ - period time in millisecond (default is 86400000).
+-   *period* - period time in millisecond (default is 86400000).
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc]: capital P for Period

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 9]
```diff
@@ -176,16 +176,16 @@ To use this policy, you have to precise the following parameters:
...
 
--   +schedulerURL+ - URL of the Scheduler
+-   *schedulerURL* - URL of the Scheduler, e.g. pnp://mymachine:64738, pamr://0, etc
 
--   +schedulerCredentialsPath+ - Path to the credentials used for
+-   *schedulerCredentialsPath* - Path to the credentials used for
     scheduler authentication.
 
--   +idleTime+ - idle time in millisecond to wait before removing all
+-   *idleTime* - idle time in millisecond to wait before removing all
```
Comment (luispinedamx) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc]: capital I in "Idle"

**Comment by fviale** [comment]
Retest this please

**Pull request merged**